src/app/base/components/ThemedRadioButton/ThemedRadioButton.tsx

```diff
@@ -0,0 +1,36 @@
+import type { ReactNode } from "react";
+
+import classNames from "classnames";
+
+export type Props = {
+  checked?: boolean;
+  className?: string;
+  colour?: string;
+  name?: string;
+  label?: ReactNode;
+} & React.PropsWithoutRef<JSX.IntrinsicElements["input"]>;
+
+const ThemedRadioButton = ({
```

Comment:
It doesn't seem that this component could be used in places other than the the theme setting? Might be good to rename it to reflect that, and move outside of the general components directory to where it's being used (in the GeneralForm folder).